### PR TITLE
feat(setup): S3 configuration

### PR DESCRIPTION
**Pull request by lwander**

### CODE REVIEW

**Comment by lwander** [comment]
ping

File: setup/storage/s3.md [hunk 1]
```diff
@@ -5,4 +5,39 @@ sidebar:
   nav: setup
 ---
 
-Documentation for S3 setup with Halyard is coming soon!
+{% include toc %}
```
Comment (ttomsu) [inline setup/storage/s3.md]: Remove TOC for short pages

File: setup/storage/s3.md [hunk 2]
```diff
@@ -5,4 +5,39 @@ sidebar:
...
+
+## Prerequisites
+
+You need an [AWS Account](https://aws.amazon.com/account/), and an associated
+access key (secret access key & access key id). 
+
+## Editing your Storage Settings
+
+Halyard will create a bucket for you if the bucket you specify doesn't exist
+yet, or if you don't specify one at all. However, if this is the case, you need
```
Comment (ttomsu) [inline setup/storage/s3.md]: The phrase "if this is the case" follows an "A or B" statement, making it ambiguous to what "this" is referring to. I suggest restating to: "If you specify the bucket, ensure to also supply the bucket's region"

File: setup/storage/s3.md [hunk 3]
```diff
@@ -5,4 +5,39 @@ sidebar:
...
+to supply the bucket's region (e.g. `REGION=us-west-1`).
+
+To configure the S3 settings, run the following (notice, do not supply the
+value of `--secret-access-key` on the command line, you will be prompted to
+enter the value on STDIN once the command has started running):
+
+```bash
+hal config storage s3 edit --secret-access-key \
+    --secret-key-id $YOUR_SECRET_KEY_ID \
```
Comment (ttomsu) [inline setup/storage/s3.md]: Doesn't the key ID generally come first? I'd keep the standard userID -> password ordering.

**Review by ttomsu — APPROVED** [review]

File: setup/storage/s3.md (hunk 2 above)
Comment (lwander) [reply setup/storage/s3.md]: It applies to both cases, I'll find an alternate wording

**Pull request merged**